usb/esp_tinyusb/CMakeLists.txt

```diff
@@ -27,9 +27,13 @@ if(CONFIG_TINYUSB_CDC_ENABLED)
     list(APPEND srcs
         "cdc.c"
         "tusb_cdc_acm.c"
+        )
+if(CONFIG_VFS_SUPPORT_IO)
```

Comment:
Nitpick: Probably, CONFIG_VFS_SUPPORT_IO_STREAMS would be little bit precisely. 
But it's up to you, indeed.

Reply:
This is coming from VFS component https://github.com/espressif/esp-idf/blob/master/components/vfs/Kconfig#L3

So we cannot change this :(

Reply:
CONFIG_VFS_SUPPORT_IO is a TOP level config option for VFS feature.

Reply:
OK, I see.
I am not insisting. :slightly_smiling_face: